### DISCUSSION TITLE
The NVDA browser nav addon should be implimented into NVDA.

### DISCUSSION THREAD

**Discussion by audiogamer22** [nvaccess/nvda#15889] [Ideas]
This is such a useful addon that enhances the navigation of webpages, for example; I can find tabs on youtube with y/shift plus y, I believe this will really help people when it comes to websites changing their layouts and making it more confusing than it has to be to navigate the web.

**Comment by josephsl** [comment]
Hi, if I read what’s new document right, starting from NVDA 2023.3, you can assign a gesture to move to next or previous tab (unassigned by default). Thanks.
